frontend/packages/kubevirt-plugin/src/selectors/vm-template/advanced.ts

```diff
@@ -49,6 +56,14 @@ export const getTemplatesLabelValues = (templates: TemplateKind[], label: string
 export const getTemplateFlavors = (templates: TemplateKind[]) =>
   getTemplatesLabelValues(templates, TEMPLATE_FLAVOR_LABEL);
 
+export const getTemplateHostname = (template: TemplateKind) => {
+  const vm = selectVM(template);
+  const cloudInitVolume = getVolumes(vm).find((vol) => !!getVolumeCloudInitUserData(vol));
```

Comment:
sorry, forgot there is a function for that: `getCloudInitVolume`

Reply:
aaand nobody is using it :) and the implementation is wrong. It should just check for the `cloudInitNoCloud`